test/JDBC/expected/BABEL-736.out

```diff
@@ -0,0 +1,245 @@
+SELECT degrees(10)
+GO
+~~START~~
+int
+572
+~~END~~
+
+
+SELECT degrees(NULL)
+GO
+~~START~~
+float
+<NULL>
+~~END~~
+
+
+SELECT degrees(9999*9999)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Arithmetic overflow error converting expression to data type int)~~
+
+
+SELECT degrees(9999.0 * 9999.0)
+GO
+~~START~~
+float
+5.72843209301375E9
+~~END~~
+
+
+
+---------------------------------------------------
+--TINYINT__TEST_CASES_FOR_DEGREES
+--Positive_value_test_case
+SELECT degrees(CAST(1 AS TINYINT));
+GO
+~~START~~
+int
+57
+~~END~~
+
+
+--Normal_value_test_case
+SELECT degrees(CAST(4 AS TINYINT));
+GO
+~~START~~
+int
+229
+~~END~~
+
+
+SELECT degrees(CAST(255 AS TINYINT));
+GO
+~~START~~
+int
+14610
+~~END~~
+
+
+--Null_value_test_case
+SELECT degrees(CAST(NULL AS TINYINT));
+GO
+~~START~~
+int
+<NULL>
+~~END~~
+
+
+--Trigger_an_error_message_integer_out_of_range
+SELECT degrees(CAST(256 AS TINYINT));
+GO
+~~START~~
+int
+~~ERROR (Code: 220)~~
+
+~~ERROR (Message: value for domain tinyint violates check constraint "tinyint_check")~~
+
+
+
+---------------------------------------------------
+--SMALLINT__TEST_CASES_FOR_DEGREES
+--Positive_value_test_case
+SELECT degrees(CAST(10 AS SMALLINT));
+GO
+~~START~~
+int
+572
+~~END~~
+
+
+--Negative_value_test_case 
+SELECT degrees(CAST(-10 AS SMALLINT));
+GO
+~~START~~
+int
+-572
+~~END~~
+
+
+--Normal_value_test_case
+SELECT degrees(CAST(5680 AS SMALLINT));
+GO
+~~START~~
+int
+325440
+~~END~~
+
+
+SELECT degrees(CAST(32767 AS SMALLINT));
+GO
+~~START~~
+int
+1877410
+~~END~~
+
+
+--Null_value_test_case
+SELECT degrees(CAST(NULL AS SMALLINT));
+GO
+~~START~~
+int
+<NULL>
+~~END~~
+
+
+--Trigger_an_error_message_integer_out_of_range
+SELECT degrees(CAST(32768 AS SMALLINT));
+GO
+~~ERROR (Code: 220)~~
+
+~~ERROR (Message: smallint out of range)~~
```

Comment:
why do you still see this error ? 
Please make sure your test could run without problem